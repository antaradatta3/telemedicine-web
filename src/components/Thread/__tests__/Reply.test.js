import React from 'react';
import { mount } from 'enzyme';

import Reply from '../Reply';

describe('Component: Reply', () => {
  const props = {
    reply: {
      data: {
        ups: 100,
        author: 'sample',
        downs: 0,
        replies: {
          data: {
            children: [],
          },
        },
      },
    },
    counter: 1,
  };

  it('Reply matches snapshot', () => {
    const component = mount(<Reply {...props} />);
    expect(component).toMatchSnapshot();
  });
});
