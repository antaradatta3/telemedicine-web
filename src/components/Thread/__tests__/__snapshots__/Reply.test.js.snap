// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Reply Reply matches snapshot 1`] = `
<Reply
  counter={1}
  reply={
    Object {
      "data": Object {
        "author": "sample",
        "downs": 0,
        "replies": Object {
          "data": Object {
            "children": Array [],
          },
        },
        "ups": 100,
      },
    }
  }
>
  <div
    className="alert text-left border-top-0"
  >
    <div>
      <small>
        »
      </small>
      reply from  
      <a
        href="https://www.reddit.com/user/sample"
      >
        sample
      </a>
       
      <small>
         on 
        Invalid Date
      </small>
    </div>
    <Badge
      count={100}
      label="Ups"
      type="success"
    >
      <span>
         
        <span
          className="badge badge-pill badge-success"
        >
          100
        </span>
         
        Ups
          
      </span>
    </Badge>
    <Badge
      count={0}
      label="Downs"
      type="danger"
    >
      <span>
         
        <span
          className="badge badge-pill badge-danger"
        >
          0
        </span>
         
        Downs
          
      </span>
    </Badge>
    <p />
    <ReactMarkdown
      astPlugins={Array []}
      escapeHtml={true}
      plugins={Array []}
      rawSourcePos={false}
      renderers={Object {}}
      skipHtml={false}
      sourcePos={false}
      transformLinkUri={[Function]}
    >
      <Root />
    </ReactMarkdown>
  </div>
</Reply>
`;
